tutorial/practicals/nemo/3_nemo_openmp/README.md

```diff
@@ -114,19 +106,7 @@ the number of MPI processes and resulting inter-process communication.)
    END DO
    ```
 
-   and the `write` statement is represented as a CodeBlock in the PSyIR:
-   ```
-    20: Loop[variable='jk', loop_type='levels']
-        ...
-        Schedule[]
-            0: Loop[variable='jj']
-               ...
-               Schedule[]
-                   0: Loop[variable='ji']
-                      ...
-                      Schedule[]
-                          0: CodeBlock[[<class 'fparser.two.Fortran2003.Write_Stmt'>]]
-   ```
+   and the `write` statement is represented as a CodeBlock in the PSyIR.
```

Comment:
Any particular reason for removing this?

Reply:
Reverted deletion